### DISCUSSION TITLE
change Username

### DISCUSSION THREAD

**Discussion by HarshadHindlekar** [community/community#38241] [Other Feature Feedback, Questions, & Ideas] [answered]
somebody guide me to change my username 

my username is bossmaker and i want to change it

because i created my account during my college time and that time i dont aware of github

**Comment by radar-blip** [comment, marked as answer]
Just go to Settings, in the left column click on Account, from there you will see this: 

![username](https://user-images.githubusercontent.com/39196767/199898316-efbddc44-03a9-4767-b13c-ddfc1e626fe5.png)

**Reply by HarshadHindlekar** [reply]
thank you so much sir.
 it works/helps.

**Reply by radar-blip** [reply]
You're welcome. Happy to help!

**Reply by jaenwawe** [reply]
Wow, things change quickly.  It is not there anymore.  Does anyone know where the change username button is?

**Reply by radar-blip** [reply]
It's still there. Only reason it might not be there is if you're using your phone.

**Reply by W1-83764-Satvik** [reply]
the button is there but not clickable.

**Reply by W1-83764-Satvik** [reply]
Observe my name. i want my name as Satvik only not W1-83764-Satvik

**Comment by EPILIASHOKKUMAR** [comment]
i need to change my username
